client/rest-high-level/src/test/java/org/elasticsearch/client/CCRIT.java

```diff
@@ -270,6 +272,20 @@ public void testAutoFollowing() throws Exception {
             getIndexSettingsAsMap("copy-logs-20200101"),
             hasEntry("index.number_of_replicas", Integer.toString(followerNumberOfReplicas)));
 
+        CreateIndexRequest createExcludedIndexRequest = new CreateIndexRequest("logs-excluded");
+        CreateIndexResponse createExcludedIndexResponse =
+            highLevelClient().indices().create(createExcludedIndexRequest, RequestOptions.DEFAULT);
+        assertThat(createExcludedIndexResponse.isAcknowledged(), is(true));
```

Comment:
I think we should create this before the `logs-20200101` index, then once we see the copy of the `logs-20200101` index, we can also verify that we do not see the excluded index.